images/manageiq-orchestrator/container-assets/entrypoint

```diff
@@ -66,7 +73,8 @@ case $? in
     pushd ${APP_ROOT}
       set -e
       REGION=${DATABASE_REGION} bin/rake db:migrate
-      REGION=${DATABASE_REGION} bin/rails runner "MiqDatabase.seed; MiqRegion.seed"
+      REGION=${DATABASE_REGION} bin/rake evm:db:seed
```

Comment:
What is the side effect of switching to this full seed from the near-primordial seed?

Reply:
Also should we prefer `rake db:seed` here?

Reply:
> What is the side effect of switching to this full seed from the near-primordial seed?

I wasn't sure what we needed to add a group correctly, so I just went for everything. I figured that we would need `MiqGroup`, `MiqUserRole`, `Tenant` at least, but couldn't be sure of what else and was running into weird issues (but a successful add) when I didn't have the right things seeded.

> Also should we prefer rake db:seed here?

Oh, they're the same .. Sure? Why do we have two?

Reply:
> Why do we have two?

I honestly can't remember 😅